### ISSUE TITLE
Keys detective doesn't show Missing/Extra keys correctly

### ISSUE THREAD

**Issue by ronnetzer** [ngneat/transloco-keys-manager#20]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

When running keys detector on a project/i18n folder I always get the same result that all the keys for all langs are Extra keys and there's nothing in Missing keys section (even when I deliberately add a key only in one lang file).

## Expected behavior

Extra keys should include only keys that are not used in the templates.
Missing keys should include keys that are in other i18n files and not in the current one.

## Minimal reproduction of the problem with instructions

Create 2 i18n files with a key that isn't in the other file and use them in the HTML

## Environment

<pre><code>
Angular version: 8.2.3
Keys Manager: 1.3.2
Node version: 10.13.0
Platform: Mac
</code></pre>

Maybe its also important to mention that my folder structure is angular's monorepo style (multiple apps and libs under projects folder).
I've tried:
`npm run i18n:find -- --project="projectName"`
`npm run i18n:find -- -p projects/projectName/src/assets/i18n`
and also `npm run i18n:find` with `rootTranslationsPath` pointing to a specific project

**Comment by shaharkazaz** [comment]
@ronnetzer can you please run the command with `DEBUG=config` and share the result?

**Comment by ronnetzer** [comment]
```bash
  config Default: { input: 'src/app', output: 'src/assets/i18n', langs: [ 'en' ], marker: 't', sort: false, defaultValue: undefined, replace: false, addMissingKeys: false, translationsPath: 'src/assets/i18n' } +0ms
  config Transloco file: { translationsPath: 'assets/i18n/', langs: [ 'en', 'he' ] } +4ms
  config Inline: { project: 'paypo-v5' } +1ms
  config Merged: { input: 'projects/paypo-v5/src/src/app', output: 'projects/paypo-v5/src/src/assets/i18n', langs: [ 'en', 'he' ], marker: 't', sort: false, defaultValue: undefined, replace: false, addMissingKeys: false, translationsPath: 'projects/paypo-v5/src/assets/i18n/', project: 'paypo-v5' } +0ms
```

**Comment by shaharkazaz** [comment]
@ronnetzer as you can see your `input` and `output` file contain double `src` since they were prefixed with the project's `sourcePath`. 
You need to provide the input and output and not use the defaults, try it and let me know 🙂 
I'll make it clearer in the docs.

**Comment by ronnetzer** [comment]
@shaharkazaz That partially solves it, Now missing keys have all the keys that are in the HTML and not in the JSONs but it doesn't include keys that are missing between the JSONs.

Also, it's a little weird that I need to override values in order to make it work correctly, especially if I don't use those features (output in this case). I think that you should handle the extra `src` when prefixing with the project's `sourcePath` on your side.
and even if those properties are not defined correctly, I still expect missing keys between JSONs to work correctly

**Comment by shaharkazaz** [comment]
@ronnetzer I'm not sure I understand what you mean by "missing keys between JSONs", can you explain?

**Comment by ronnetzer** [comment]
> This tool detects two things: First, it detects any key that exists in one of your translation files, but is missing in any of the others

**Comment by shaharkazaz** [comment]
@ronnetzer fixed in version 2.0.0
